app/components/questions.js

```diff
@@ -0,0 +1,108 @@
+// https://github.com/EnCiv/undebate-ssp/issues/48
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import CountLimitedTextInput from './count-limited-text-input'
+import Submit from './submit'
+
+export default function Questions({ electionOM, onDone }) {
+    const classes = useStyles()
+    const { electionMethods } = electionOM
+    const [questions, setQuestions] = useState([''])
+    const [error, setError] = useState('')
+    const [isValid, setIsValid] = useState(false)
+
+    const checkValid = () =>
+        !electionMethods.areQuestionsLocked() &&
+        (questions.length > 1 || (questions.length === 1 && !questions.includes('')))
+
+    useEffect(() => {
+        setIsValid(checkValid())
+    }, [isValid, electionMethods, questions])
+
```

Comment:
should depend on electionOM rather than electionMethods. electionObj and electionMethods may not change when the data changes, but electionOM always will.